## file 'MyCustomMessage.msg'

bool value_boolean
int32 value_integer
float32 value_float
string value_string
